libs/hs/qua-types/src/QuaTypes/Review.hs

```diff
@@ -84,6 +84,7 @@ instance ToJSON   ExpertReviewPost
 -- | Previous reviews of the viewed design
 data Review = Review
   { reviewUserName  :: QuaText
+  , reviewIsMine    :: !Bool -- ^ review is by currently logged in user
```

Comment:
The file is annotated with `{-# LANGUAGE Strict #-}`, which means your bang has no effect